tests/HttpClientInterception.Benchmarks/InterceptionBenchmarks.cs

```diff
@@ -155,7 +155,11 @@ public async Task<HttpContent> SerializeAsync<T>(T item)
                 }
                 catch (Exception)
                 {
+#if NETCOREAPP3_0
                     await stream.DisposeAsync();
+#else
+                    stream.Dispose();
+#endif
```

Comment:
fwiw `MemoryStream` `DisposeAsync` just calls `Dispose`

Reply:
Maybe I'll simplify it to just use the sync one everywhere as it's always a concrete `MemoryStream`.